src/actionItems.ts

```diff
@@ -10,14 +10,6 @@ export enum ActionItemCode {
    * API Keys that are too old (expired)
    */
   ApiKeyStale = 'API_KEY_STALE',
-  /**
-   * An assessment has been assigned for completion
-   */
-  AssessmentAssigned = 'ASSESSMENT_ASSIGNED',
```

Comment:
No, but I think I have to remove them here first, and then I'll update the package in the monorepo and remove them from code there.

Reply:
Ok! It may be tricky if it's the same exact enum used in the frontend and the backend, because we'll want to remove the reference from frontend first, deploy that, then backend..

Reply:
Yeah, part of the difficulty is that a lot of the FE is typed to expect the whole enum to be there, so it throws TS errors if you try to do the FE first.

Reply:
That might work... 🤔

Reply:
Does this plan in the description here sound right? https://transcend.height.app/T-34487

Reply:
Yep! Sounds perfect!!